documentation/en/source/pages/command-line/migrate.rst

```diff
@@ -0,0 +1,88 @@
+.. Copyright (c) 2007-2018 UShareSoft, All rights reserved
+
+.. _command-line-migrate:
+
+migration
+=========
+
+Manages all the migrations executed on live systems. The usage is:
+
+.. code-block:: shell
+
+	usage: hammr migration [sub-command] [options]
+
+
+Sub Commands
+------------
+
+``list`` sub-command
+~~~~~~~~~~~~~~~~~~~~
+
+Displays all the migrations for the user.
+
+``launch`` sub-command
+~~~~~~~~~~~~~~~~~~~~~~
+
+Executes an automated migration (including scan, generate and publish) of a running system, based on a yaml or json file. The options are:
```

Comment:
Just a detail, but sometimes we use "yaml" and other times "YAML", I don't know if it's normal. Idem for "json"

Reply:
let's raise a jira bug to do a fix for this throughout doc